GVRf/Framework/jni/objects/mesh_eye_pointee_jni.cpp

```diff
@@ -39,28 +36,16 @@ Java_org_gearvrf_NativeMeshEyePointee_setMesh(JNIEnv * env,
 JNIEXPORT jlong JNICALL
 Java_org_gearvrf_NativeMeshEyePointee_ctor(JNIEnv * env,
         jobject obj, jlong jmesh) {
-    std::shared_ptr<Mesh> mesh =
-            *reinterpret_cast<std::shared_ptr<Mesh>*>(jmesh);
-    return reinterpret_cast<jlong>(new std::shared_ptr<MeshEyePointee>(
-            new MeshEyePointee(std::move(mesh))));
-}
-
-JNIEXPORT jlong JNICALL
-Java_org_gearvrf_NativeMeshEyePointee_getMesh(JNIEnv * env,
-        jobject obj, jlong jmesh_eye_pointee) {
-    std::shared_ptr<MeshEyePointee> mesh_eye_pointee =
-            *reinterpret_cast<std::shared_ptr<MeshEyePointee>*>(jmesh_eye_pointee);
-    std::shared_ptr<Mesh> mesh = mesh_eye_pointee->mesh();
-    return reinterpret_cast<jlong>(new std::shared_ptr<Mesh>(mesh));
+    Mesh* mesh = reinterpret_cast<Mesh*>(jmesh);
```

Comment:
mesh() also becomes dead codes here which is similar to camera_rig.left/right_camera()